trans-calc.py

```diff
@@ -207,6 +211,34 @@ def clear_fields(self) -> None:
 
         self.matrix_entries[0].focus_set()
 
+    def copy_full_price(self) -> None:
+    full_price = self.full_price_sv.get()
+    self.root.clipboard_clear()
+    self.root.clipboard_append(full_price)
+    self.root.update()
+
+    # Optionally, display a message to indicate that the value has been copied
+    copied_label = ttk.Label(self.full_price_frame, text="Full price copied!", style="Success.TLabel")
+    copied_label.grid(sticky="w", column=2, row=0, padx=(10, 0), pady=(40, 0))
+    self.root.after(2000, lambda: copied_label.grid_forget())
+
+    def copy_total_words(self) -> None:
+    total_words = self.total_words_sv.get()
+    self.root.clipboard_clear()
+    self.root.clipboard_append(total_words)
+    self.root.update()
+
+    # Optionally, display a message to indicate that the value has been copied
+    copied_label = ttk.Label(self.secondary_frame, text="Total words copied!", style="Success.TLabel")
+    copied_label.grid(column=2, row=0, padx=(10, 0), pady=(0, 0))
+    self.root.after(2000, lambda: copied_label.grid_forget())
+
+    def bind_hotkeys(self) -> None:
+    #Hotkey can be adjusted as preferred 
+    self.root.bind("<Control-Alt-c>", lambda e: self.clear_fields())
+    self.root.bind("<Control-Alt-f>", lambda e: self.copy_full_price())
+    self.root.bind("<Control-Alt-w>", lambda e: self.copy_total_words())
```

Comment:
The above methods throw `IndentationError`. Please check the indentation under the method definitions.